### PR TITLE
add docs

### PR DESCRIPTION
**Pull request by itajaja**

### CODE REVIEW

File: docs/src/components/HomepageFeatures.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* stylelint-disable docusaurus/copyright-header */
+
+.features {
+  display: flex;
+  align-items: center;
+  padding: 2rem 0;
+  width: 100%;
+}
+
+.featureSvg {
```
Comment (AaronBuxbaum) [inline docs/src/components/HomepageFeatures.module.css:10]:
```suggestion
.feature-svg {
```
css convention

File: docs/src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      </div>
+    </header>
+  );
+}
+
+export default function Home() {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+    <Layout
+      title={`${siteConfig.title}`}
```
Comment (AaronBuxbaum) [inline docs/src/pages/index.js:39]:
```suggestion
      title={siteConfig.title}
```

File: docs/src/pages/demo.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+export default function Demo() {
+  return (
+    <>
+      <iframe
+        src={useBaseUrl("/try.html")}
```
Comment (c0state) [inline docs/src/pages/demo.js:9]: How come this was named `try`? Seems odd...

Comment (itajaja) [reply docs/src/pages/demo.js:9]: Try as in "try the demo"  😅 no the best name

File: docs/src/components/HomepageFeatures.module.css (hunk 1 above)
Comment (itajaja) [reply docs/src/components/HomepageFeatures.module.css:10]: I think I tried, no r sure this plug-in translates

**Pull request merged**